ISSUE_TEMPLATE/data-intake.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Data Intake
+about: Investigate the use of external data
+title: Data Intake
+labels: data intake
+assignees: danielolsen, rouille
+
+---
+
+### Source
+* Name: ***e.g.*** Form EIA-923
+* Author: ***e.g.*** EIA
+* Description: ***e.g.*** Electric power data -- monthly and annually -- on electricity generation, fuel consumption, fossil fuel stocks, and receipts at the power plant and prime mover level for 2016
+* Source: ***e.g.*** https://www.eia.gov
+* Exact source location: ***e.g.*** https://www.eia.gov/electricity/data/eia923/
+* Terms (if specified): ***e.g.*** https://www.eia.gov/about/copyrights_reuse.php where it is written that we may use and/or distribute any of their data, files, databases, reports, graphs, charts, and other information products that are on their website
+
+### Destination
+* Modifications to source files(s): ***e.g.*** None
+* Location: powersimdata/...
```

Comment:
We could replace `powersimdata/...` with something generic, like `YOUR_REPO/...`